_posts/2015/2015-09-14-textlint-rul-prh.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "textlint + prhで表記ゆれを検出する"
+author: azu
+layout: post
+date : 2015-09-14T18:00
+category: JavaScript
+tags:
+    - textlint
+    - tools
+
+---
+
+## [proofread-helper](https://github.com/vvakame/prh "proofread-helper")
+
+[proofread-helper](https://github.com/vvakame/prh "proofread-helper")は@[vvakame](https://github.com/vvakame "vvakame")さんが作っている文章の校正補助ツールです。([Tech Booster](http://techbooster.org/ "Tech Booster")とかの執筆で使っている?らしいです)
```

Comment:
fixed in https://github.com/efcl/efcl.github.io/commit/b8ceed167d54e0cfe45f2668b948e1f43059725c